replica/compaction_group.hh

```diff
@@ -43,6 +43,8 @@ class compaction_group {
     // have not been deleted yet, so must not GC any tombstones in other sstables
     // that may delete data in these sstables:
     std::vector<sstables::shared_sstable> _sstables_compacted_but_not_deleted;
+    // sstables that require cleanup
+    std::unordered_set<sstables::shared_sstable> _cleanup_sstables_set;
```

Comment:
How is this persisted across restarts?

Reply:
it isn't, it should be rebuilt.
the workaround is to rerun cleanup compaction if a node restarts before cleanup is done.
It's a pre-existing issue.